pkg/ccl/backupccl/restore_span_covering.go

```diff
@@ -185,6 +212,31 @@ func makeSimpleImportSpans(
 	return cover
 }
 
+// createIntroducedSpanFrontier creates a span frontier that tracks the end time
+// of the latest incremental backup of each introduced span in the backup chain.
+// See ReintroducedSpans( ) for more information. Note: this function assumes
+// that manifests are sorted in increasing EndTime.
+func createIntroducedSpanFrontier(
+	manifests []backuppb.BackupManifest, asOf hlc.Timestamp,
+) (*spanUtils.Frontier, error) {
+	introducedSpanFrontier, err := spanUtils.MakeFrontier(roachpb.Span{})
+	if err != nil {
+		return nil, err
+	}
+	for i, m := range manifests {
+		if i == 0 {
```

Comment:
confirming this is because we want to skip the full?

Reply:
yes. Note that this step doesn't do much because the full backup shouldn't have introduced spans anyway.